src/core/dev/ib_ctx_handler.cpp

```diff
@@ -73,30 +73,10 @@ ib_ctx_handler::ib_ctx_handler(struct ib_ctx_handler_desc *desc)
     }
 
     m_p_ibv_context = NULL;
-#ifdef DEFINED_DPCP
     m_p_adapter = set_dpcp_adapter();
-    if (NULL == m_p_adapter)
-#endif /* DEFINED_DPCP */
-    {
-#if defined(DEFINED_ROCE_LAG)
```

Comment:
[question] Does it mean that we don't support ROCE_LAG in any way? Or support is transparent?

Reply:
It should be transparent. But officially we do not support ROCE LAG.